core/src/main/java/org/mule/runtime/core/internal/exception/AbstractExceptionListener.java

```diff
@@ -119,15 +118,6 @@ private Throwable getCause(Exception ex) {
     return ex.getCause() instanceof TypedException ? ex.getCause().getCause() : ex.getCause();
   }
 
-  protected void closeStream(Message message) {
```

Comment:
StreamCloser is still used in batch, so i left infrastructure in place. Also when we implement auto-closing of streams for non-repeatable streams we may want to use StreamCloserService.